src/server/server.ts

```diff
@@ -538,7 +538,15 @@ namespace ts.server {
 
             fs.stat(watchedFile.fileName, (err: any, stats: any) => {
                 if (err) {
-                    watchedFile.callback(watchedFile.fileName, FileWatcherEventKind.Changed);
+                    if (err.code === "ENOENT") {
+                        if (watchedFile.mtime.getTime() !== 0) {
+                            watchedFile.mtime = new Date(0);

```

Comment:
So we're using `0` as a special sigil value? Why not use `undefined` for that purpose?

Reply:
@sheetalkamat bump

Reply:
hmm. I was using same logic that was already present. But I will look into it.